src/main/java/com/google/sps/servlets/DashboardServlet.java

```diff
@@ -14,62 +14,125 @@
 import com.google.appengine.api.datastore.Query.Filter;
 import com.google.appengine.api.datastore.Query.FilterPredicate;
 import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.CompositeFilterOperator;
 import com.google.sps.servlets.Property;
 import java.lang.reflect.Field; 
 import java.util.HashMap;
+import com.google.sps.servlets.RequestUtils;
 
 @WebServlet("/dashboard")
 public class DashboardServlet extends HttpServlet {
 
+  private final long MILLISECONDS_IN_DAY = 86400000;
+  private final String DEFAULT_START_TIMESTAMP = "0";
+  private final String DEFAULT_END_TIMESTAMP = String.valueOf(System.currentTimeMillis());
```

Comment:
This will make it so that the default end time is determined whenever you first run the app.  I don't think youll be able to use a constant for this

Reply:
Changed it!